pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -178,6 +181,15 @@ def set_world_ranks(self):
         self.world_size = self.num_nodes * self.num_processes
 
     def configure_ddp(self):
+
+        # todo: PyTorch 1.7.0 DDP introduces ``self.reducer._rebuild_buckets()``` breaking manual_optimization
+        if _PYTORCH_GREATER_EQUAL_THAN_1_7_0 and not self.lightning_module.automatic_optimization:
```

Comment:
Do we have to make all these changes to ddp spawn as well?
Since this also uses ddp just with spawned processes...

Reply:
Oh yeah, you are right ! My bad.